src/components/atoms/NavLink.cy.tsx

```diff
@@ -0,0 +1,18 @@
+import { DashboardIcon } from '../appShell/icons/DashboardIcon';
+import { NavLink } from './NavLink';
+
+describe('NavLink', () => {
+  it('to render and have right text', () => {
+    cy.mount(
+      <NavLink
+        links={[
+          {
+            label: 'داشبورد',
+            icon: <DashboardIcon w="24" h="24" />,
+          },
+        ]}
+      />,
+    );
+    cy.get('[data-test-id="nav-link"]').should('have.text', 'داشبورد');
```

Comment:
testing the text is not really a good practice so let's not do that 
a Navlink

maybe we can ignore testing the navlink since we test their functionality in the app shell and since navlink is actually a series of navlinks not a single navlink so we can't really test single navlink unless we change navlink to be single navlink


@ASafaeirad @mojtabaesl what you guys say?